dbt/include/singlestore/macros/materializations/incremental/incremental.sql

```diff
@@ -3,5 +3,44 @@
 {% endmacro %}
 
 {% macro singlestore__get_delete_insert_merge_sql(target, source, unique_key, dest_columns, incremental_predicates) %}
-    {% do return (default__get_delete_insert_merge_sql(target, source, unique_key, dest_columns, incremental_predicates)) %}
+
```

Comment:
please add a comment on what's changed compared to `default__get_delete_insert_merge_sql`